.travis.yml

```diff
@@ -1,64 +1,67 @@
+  # starting nginx
+  - mkdir /home/travis/miniconda3/envs/qiita/var/run/nginx/
+  - wget https://github.com/biocore/qiita/archive/dev.zip -O qiita-dev.zip
+  - unzip qiita-dev.zip
+  - nginx -c ${PWD}/qiita-dev/qiita_pet/nginx_example.conf
   - qiita-env make --no-load-ontologies
   - source deactivate
   # installing qp-shogun
-  - conda create --yes -n qp-shogun python=$PYTHON_VERSION pip nose flake8
+  - conda config --add channels defaults
+  - conda config --add channels bioconda
+  - conda config --add channels conda-forge
+  - conda create --yes -n qp-shogun -c knights-lab shogun=1.0.7 biom-format samtools bedtools pigz
   - source activate qp-shogun
-  - pip install -U pip
-  - conda install --yes pigz
-  - conda install --yes -c bioconda bowtie2 fastqc=0.11.5 samtools bedtools
+  - pip install -U pip pip nose flake8
   - pip install sphinx sphinx-bootstrap-theme nose-timer codecov biopython
   - pip install https://github.com/qiita-spots/qiita_client/archive/master.zip
   - pip install atropos
-  - pip install git+https://github.com/knights-lab/SHOGUN.git
-  - export QIITA_SERVER_CERT=$PWD/miniconda3/envs/qiita/lib/python2.7/site-packages/qiita_core/support_files/server.crt
-  # just making sure we have the latest flake8
-  - pip install -U flake8
+  - export QIITA_SERVER_CERT=$HOME/miniconda3/envs/qiita/lib/python3.6/site-packages/qiita_core/support_files/server.crt
   - pip install .
   - export QC_FILTER_DB_DP=$PWD/qp_shogun/filter/databases/
   - export QC_SHOGUN_DB_DP=$PWD/qp_shogun/shogun/databases/
   - for file in $(ls $QC_SHOGUN_DB_DP); do tar xvjf $QC_SHOGUN_DB_DP$file -C $QC_SHOGUN_DB_DP; done
-  # installing other dependencies
-  - wget --quiet https://bitbucket.org/biobakery/metaphlan2/get/default.zip
-  - unzip -qq default.zip 'biobakery-metaphlan2*/utils/*' 'biobakery-metaphlan2*/metaphlan2.py'
-  - wget --quiet https://github.com/biocore/oecophylla/blob/dev/test_data/test_dbs/mp2_mini.tar.gz?raw=true -O mp2_mini.tar.gz
-  - tar zxf mp2_mini.tar.gz
-  - wget https://github.com/knights-lab/UTree/releases/download/v2.0c/utree-search_gg
-  - chmod 755 utree-search_gg
-  - mv utree-search_gg $CONDA_PREFIX/bin
-  - wget https://github.com/knights-lab/BURST/releases/download/v0.99.7f/burst_linux_DB15
+  # shogun is currently installing an old version of burst so let's install the correct one
+  - wget https://github.com/knights-lab/BURST/releases/download/v0.99.8/burst_linux_DB15
   - chmod 755 burst_linux_DB15
   - mv burst_linux_DB15 $CONDA_PREFIX/bin/burst15
+  # making sure we have the correct software versions
+  - shogun_version=`shogun --version`
+  - bowtie2_version=`bowtie2 --version`
+  - burst15_version=`burst15 -h | grep BURST`
+  - utree_version=`(utree-search_gg || true)`
+  - if [[ $shogun_version != *"1.0.7"* ]]; then echo "wrong shogun version", $shogun_version; exit 1; fi
+  - if [[ $bowtie2_version != *"2.3.5"* ]]; then echo "wrong bowtie2 version", $bowtie2_version; exit 1; fi
```

Comment:
Just FYI: Bowtie2 is backward- / forward- compatible. Either 2.3.2 or 2.3.6 (in the future) should just work. Whereas the versions for BURST and UTree are very strict.